test/sql/udf_function/test_argumented_UDF_function.cpp

```diff
@@ -0,0 +1,400 @@
+#include "catch.hpp"
```

Comment:
Lowercase in filenames please :) avoids some headaches on case-insensitive file systems.